src/synthesizer/line.py

```diff
@@ -9,30 +9,130 @@
 from synthesizer.conversions import lnu_to_llam
 from synthesizer.units import Quantity
 from synthesizer import exceptions
+from unyt import Angstrom
 
 
 def get_line_id(id):
     """
     A class representing a spectral line or set of lines (e.g. a doublet)
 
     Arguments
-    ----------
-    id : str, list, tuple
-        a str, list, or tuple containing the id(s) of the lines
+        id (str, list, tuple)
+            a str, list, or tuple containing the id(s) of the lines
 
     Returns
-    -------
-    string
-        string representation of the id
-
-    """ "H 1 6564.62A"
+        id (str)
+            string representation of the id
+    """
 
     if isinstance(id, list):
         return ",".join(id)
     else:
         return id
 
 
+# dictionary of common line labels to use by default
+special_line_labels = {
+    'O 2 3726.03A,O 2 3728.81A': '[OII]3726,3729',
+    'H 1 4862.69A': r'H\beta',
+    'O 3 4958.91A,O 3 5006.84A': '[OIII]4959,5007',
+    'H 1 6564.62A': r'H\alpha',
+    'O 3 5006.84A': '[OIII]5007',
+    'N 2 6583.45A': '[NII]6583',
+    'N 2 6583.45A': '[NII]6583',
+}
+
+
+def get_line_label(line_id):
```

Comment:
Why have these methods been moved from a `LineCollection` to independent methods?

Reply:
They feel more natural outside and it gives you option of making the label even if you don't have a line collection.

Reply:
Can a LineCollection retain wrappers around these to maintain the previous behaviour?

Reply:
I'm not sure this is useful but I've added wrappers.